malcolm/core/numberarraymeta.py

```diff
@@ -0,0 +1,49 @@
+from collections import OrderedDict
+
+import numpy
+
+from malcolm.core.scalarmeta import ScalarMeta
+from malcolm.core.serializable import Serializable
+from malcolm.compat import base_string
+
+
+@Serializable.register("malcolm:core/NumberArrayMeta:1.0")
+class NumberArrayMeta(ScalarMeta):
+    """Meta object containing information for an array of numerical values"""
+
+    def __init__(self, name, description, dtype_list):
+        super(NumberArrayMeta, self).__init__(name, description)
+        self.dtype_list = dtype_list
+
+    def validate(self, value):
+        if value is None:
```

Comment:
The output should be a numpy array.

Something like this:
https://github.com/dls-controls/malcolm/blob/master/malcolm/core/vtype.py#L84